### ISSUE TITLE
[BUG] julia-client does not install or uninstall while ink is enabled

### ISSUE THREAD

**Issue by benninkrs** [JunoLab/Juno.jl#295]
Please search existing issues to avoid duplicates.

## Details
- Atom version: 1.36.0 x64
- Julia version: 1.0.0
- OS: Windows 10 (x86_64-w64-mingw32)
- Package versions:
  - Atom.jl: 0.8.4
  - julia-client: 0.8.3
  - ink: 0.10.4

### Steps to reproduce

1. Wait until new version of Julia-client is available
2. In Atom settings/install, click install on Julia-client.
3. Get busy indicator forever
4. Close Atom.
5. Restart Atom.
6. Atom shows new version of Julia-client is installed, but I'm suspicious.
7. Click uninstall on Julia-client.
8. Julia-client is still shown as installed.  Get error message `Failed to delete julia-client: EPERM: operation not permitted, unlink 'C:\Users\Ryan\.atom\packages\julia-client\node_modules\node-pty-prebuilt\build\Release\pty.node'`
9. Close Atom
10. Restart Atom
11. Disable Ink
12. Click uninstall on Julia-client. It uninstalls.
13. Click Install on Julia-client.  It installs the latest version.
14. Enable ink.
15. Everything seems ok.

This behavior has been consistent for the last several updates of Julia-client (since about 0.8.0).

**Comment by pfitzseb** [comment]
Yeah, that's rather unfortunate. Until this is fixed the recommended update procedure on Windows is something along the lines of http://docs.junolab.org/latest/man/faq/#Juno-doesn't-work-properly-after-some-Atom-packages-were-updated.-What-do-I-do?-1.
